app/component/map/tile-layer/Covid19OpeningHours.js

```diff
@@ -61,7 +61,7 @@ class Covid19OpeningHours {
 
   // eslint-disable-next-line no-unused-vars
   getIcon = category => {
-    return 'poi_other';
+    return `poi_${category}` || 'poi_other';
```

Comment:
This doesn't do what you intent. If `category is null, this would return `category_null`. You need to place the `||` inside the `${}`.